jhputnam/mailroom_with_class.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python3
+
+
+"""Mailroom script to send thank you emails to donors"""
+
+import sys
+import pickle
+from mailroom_helper import *
```

Comment:
`*` import is considered bad form.  When possible, always try to import something specific, especially since this helper file has a class and isn't just a collection of `def`s